### PR TITLE
Design document on adding Recipe Information to the Resource

### PR DESCRIPTION
**Pull request by ytimocin**

### CODE REVIEW

File: resources/2023-10-recipe-details.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
+# Title
```
Comment (rynowak) [inline resources/2023-10-recipe-details.md:1]: Please update the title

File: resources/2023-10-recipe-details.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+### Goals
+
+* The goal is to add the recipe information to the resource and the data model level so that when the resource details is requested, recipe information will also be available.
+
+### Non goals
+
+* Everything other than adding the recipe information to the resource and data model level.
+* No changes to any of the existing APIs. The APIs will automatically pick up the changes.
+
+### User scenarios (optional)
```
Comment (rynowak) [inline resources/2023-10-recipe-details.md:34]: Should this also include the application graph API?

Comment (ytimocin) [reply resources/2023-10-recipe-details.md:34]: I think we have another design document for the Application Graph.

Comment (ytimocin) [reply resources/2023-10-recipe-details.md:34]: https://github.com/radius-project/design-notes/blob/main/resources/2023-10-app-graph.md

**Review by rynowak — APPROVED** [review]

**Review by kachawla — APPROVED** [review]

**Pull request merged**